src/pages/en/guides/deploy/kinsta.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Deploy your Astro Site to Kinsta Application Hosting
+description: How to deploy your Astro site to the web on Kinsta Application Hosting.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can use [Kinsta Application Hosting](https://kinsta.com/application-hosting/) to host an Astro site on their cloud hosting.
+
+:::tip[Looking for an example?]
+Check out [the official Kinsta Application Hosting Starter project for Astro](https://github.com/kinsta/hello-world-astro)!
+:::
+
+## Configuring your Astro project
+To host your project on **Kinsta Application Hosting**, you need to:
+- make sure your `package.json` has a `name` field.
+- have a script called `build` in your `package.json`. (Your Astro project should already include this.)
+- install the [`serve`](https://www.npmjs.com/package/serve) package and set the `start` script to `serve dist/`.
+
+Here are the necessary lines in your `package.json` file:
+```json title="astro.config.mjs" {2,6} ins={12} "serv dist/"
+{
+  "name": "@example/blog",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "serve dist/",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "dependencies": {
+    "astro": "^1.6.10",
+    "serve": "^14.0.1"
+  },
+}
+```
```

Comment:
``````suggestion
## Configuring your Astro project
To host your project on **Kinsta Application Hosting**, you need to:
- Include a `name` field in your `package.json`. (This can be anything, and will not affect your deployment.)
- Include a `build` script in your `package.json`. (Your Astro project should already include this.)
- Install the [`serve`](https://www.npmjs.com/package/serve) package and set the `start` script to `serve dist/`.

Here are the necessary lines in your `package.json` file:
```json title="astro.config.mjs" {2,6} ins={12} "serv dist/"
{
  "name": "anything", // This is required, but the value does not matter.
  "scripts": {
    "dev": "astro dev",
    "start": "serve dist/",
    "build": "astro build",
    "preview": "astro preview",
    "astro": "astro"
  },
  "dependencies": {
    "astro": "^1.6.10",
    "serve": "^14.0.1"
  },
}
```
``````
OK, given `name`-gate, do you think this makes sense/is helpful enough?  😅

Reply:
Perfect :)